### DISCUSSION TITLE
Floating Timeline

### DISCUSSION THREAD

**Discussion by Solarunit** [tomasklaen/uosc#921] [Ideas]
I have this idea for a while - Floating Timeline option, were there is a small gap at the bottom but height of Timeline should not change or it can be even more compact.  Similar to Volume but more narrow and with minimal gap. The gap area at the bottom should stay clickable. In this mode we can reveal Timeline like Volume with fade-in/fade-out animation.

**Comment by tomasklaen** [comment]
Main reason why timeline is touching all 3 sides is so that when you're in fullscreen, you can just throw the cursor against the edge of the screen and start seeking, no aiming necessary. I enjoy this UX a lot, so I won't be detaching it.

**Reply by christoph-heinrich** [reply]
The floating taskbar in kde plasma detects clicks up to the edge of the screen, even though it's technically outside of the bar. The same could be done here, and is already done for window controls.

Not that I support this idea, just throwing it out there.

**Reply by Solarunit** [reply]
I totally agree with this, I mentioned it: 

>The gap area at the bottom should stay clickable.

Otherwise it would be less usable.
